modules/otel-config-instrumentation.adoc

```diff
@@ -98,6 +98,58 @@ spec:
 
 |===
 
+[id="otel-instrumentation-options_exporter_{context}"]
+== Exporter configuration
+
+The `Instrumentation` custom resource configures only OTLP exporter:
+
+.Sample exporter configuration
+[source,yaml]
+----
+  exporter:
+    endpoint: https://production-collector.observability.svc.cluster.local:4317  # <1>
+    tls:
+      secretName: serving-certs # <2>
```

Comment:
@pavolloffay, if you don't mind, I'll split this into two example blocks, one with a Secret and the other one with a config map.

Reply:
I think the most common use-case on OCP will be using the serving certs on the collector and configuring ca bundle on the instrumentation e.g. 

```

kubectl apply -f - <<EOF
apiVersion: v1
kind: ConfigMap
metadata:
  name: otelcol-cabundle
  namespace: tutorial-application
  annotations:
    service.beta.openshift.io/inject-cabundle: "true"
EOF
```

```
kubectl apply -f - <<EOF
apiVersion: opentelemetry.io/v1alpha1
kind: Instrumentation
metadata:
  name: my-instrumentation
spec:
  exporter:
    endpoint: https://simplest-collector.tracing-system.svc.cluster.local:4317
    tls:
      configMapName: otelcol-cabundle
      ca: service-ca.crt
EOF

```